Directory.Build.props

```diff
@@ -6,9 +6,4 @@
     <GenerateDocumentationFile>false</GenerateDocumentationFile>
     <LangVersion>10.0</LangVersion>
   </PropertyGroup>
-
-  <!-- Deterministic builds -->
-  <PropertyGroup Condition="'$(TF_BUILD)' == 'true' OR '$(GITLAB_CI)' == 'true'">
-    <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
```

Comment:
The `ContinuousIntegrationBuild` setting if I'm remembering correctly has an affect on nuget package generation (producing packages that appear as valid with the nuget package explorer) so we may need to revisit this setting later.